### DISCUSSION TITLE
kaggle 우승 솔루션 github (mmdetection)

### DISCUSSION THREAD

**Discussion by kwakjeeyoon** [boostcampaitech2/object-detection-level2-cv-06#77] [Show and tell]
### Chest-X-ray-Abnormalities-Detection 2등 솔루션
https://github.com/ZFTurbo/2nd-place-solution-for-VinBigData-Chest-X-ray-Abnormalities-Detection

### kaggle-global-wheat-detection 9등 솔루션
https://github.com/amirassov/kaggle-global-wheat-detection

**Comment by jaehwan-AI** [comment]
얼른 실험 세팅이 끝나고 도전해보고 싶네요 ㅎㅎ
좋은 정보 감사합니다!
